api/v1alpha1/hostedcluster_types.go

```diff
@@ -1643,15 +1643,15 @@ type ManagedEtcdStorageSpec struct {
 	// +optional
 	PersistentVolume *PersistentVolumeEtcdStorageSpec `json:"persistentVolume,omitempty"`
 
-	// RestoreSnapshotURL allows an optional list of URLs to be provided where
+	// RestoreSnapshotURL allows an optional URL to be provided where
 	// an etcd snapshot can be downloaded, for example a pre-signed URL
-	// referencing a storage service, one URL per replica.
+	// referencing a storage service.
 	// This snapshot will be restored on initial startup, only when the etcd PV
 	// is empty.
 	//
 	// +optional
 	// +immutable
-	RestoreSnapshotURL []string `json:"restoreSnapshotURL,omitempty"`
```

Comment:
limit the slice to len 1 seems more sensible to me, so we don't change the API and we keep our ability to change this in future without needed new field or breaking clients